scripts/declarations_diff.sh

```diff
@@ -105,6 +109,9 @@ fi |
   }'
 )
 
+set +e ## TODO, bisect the error...
```

Comment:
What's the status of this TODO? 

The other changes LGTM so feel free to merge if it's not a big deal in your opinion.
bors d+

Reply:
I have tried a bit to understand why activiating `set -e` yields an error, but to no success.
It would be great to understand and fix this - but this PR is already an improvement, so I will merge it now.

Reply:
Thanks for asking and the review!